salt/modules/saltcheck.py

```diff
@@ -139,6 +166,26 @@ def run_test(**kwargs):
         return "Test must be a dictionary"
 
 
+def state_apply(state_name, **kwargs):
+    '''
+    Apply state.apply with given state and pillars to set up test data
+
+    :param str state_name: the name of a user defined state
+    :param dict kwargs: optional keyword arguments
```

Comment:
I think this could be improved to explicitly state that these arguments will be passed to `state.apply` and that the user should reference that module for a list of possible arguments to use.